specs/signature-specification.md

```diff
@@ -95,13 +29,26 @@ The Notary Project signature supports the following envelope formats:
 
 The Notary Project signature payload is a JSON document with media type `application/vnd.cncf.notary.payload.v1+json` and has following properties.
 
-- `targetArtifact` : Required property whose value is the descriptor of the target artifact manifest that is being signed. Only [OCI descriptor][oci-descriptor] is supported.
+- `targetArtifact` : Required property whose value is the descriptor of the target artifact manifest that is being signed.
+- For OCI artifacts, this MUST be a valid [OCI descriptor][oci-descriptor].
   - Descriptor MUST contain `mediaType`, `digest`, and `size` fields.
-  - Descriptor MAY contain `annotations` and if present it MUST follow the [annotation rules][annotation-rules]. The Notary Project signature uses annotations for storing both Notary Project specific and user defined metadata. The prefix `io.cncf.notary` in annotation keys is reserved for use in Notary Project signature and MUST NOT be used outside this specification.
+  - Descriptor MAY contain `annotations` and if present it MUST follow the [annotation rules][annotation-rules].
   - Descriptor MAY contain `artifactType` field for the `config.mediaType` of OCI image manifests.
+- For Blob artifacts, the descriptor MUST describe the blob that is being signed
```

Comment:
Can we say blob artifacts also uses OCI descriptors?

Reply:
Can you point out the differences?

Reply:
1. mediaType doesn't necessarily have to be `application/vnd.oci.image.manifest.v1+json`
2. There is no `artifactType`

Reply:
> 1. mediaType doesn't necessarily have to be `application/vnd.oci.image.manifest.v1+json`

[OCI descriptor](https://github.com/opencontainers/image-spec/blob/main/descriptor.md) does not restrict the media type. It can be literally anything as long as it complies with [RFC 6838](https://datatracker.ietf.org/doc/html/rfc6838).

> There is no `artifactType`

`artifactType` is an optional field in [OCI descriptor](https://github.com/opencontainers/image-spec/blob/main/descriptor.md).

Reply:
for OCI artifact registered algorithm is restricted to SHA256 and SHA512

Yes I agree we should use `OCI descriptor` with some tweaks.